pkg/get/get.go

```diff
@@ -284,3 +284,35 @@ func GetBinaryName(tool *Tool, os, arch, version string) (string, error) {
 
 	return "", errors.New("BinaryTemplate is not set")
 }
+
+// UserTools checks if name of binaries passed as argument exists
+func UserTools(tools Tools, toolArgs []string, version string) (Tools, error) {
```

Comment:
`UserTools` is not a descriptive name, what does it do?

What name should it have that describes the responsibility it has?